packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,19 @@
 const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
+  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation',
+  CREDITS_TERM_AND_CONDITIONS: '/credits-purchase/credits-terms-and-conditions',
```

Comment:
Use `CREDITS_PURCHASE_TERMS_AND_CONDITIONS`

Also the URL should be `/credits-purchase/confirm-terms-conditions`